webdriver/src/com/blazemeter/jmeter/webdriver/phantomjs/PhantomJSDriverConfig.java

```diff
@@ -34,6 +36,13 @@ protected Capabilities createCapabilities() {
         capabilities.setCapability(CapabilityType.PROXY, createProxy());
         capabilities.setCapability(PhantomJSDriverService.PHANTOMJS_EXECUTABLE_PATH_PROPERTY,
                 getPhantomJsExecutablePath());
+        String cliArgs = JMeterContextService.getContext().getVariables().get("phantomjs.cli.args");

```

Comment:
BTW, I saw new commit you merged. It adds a possibility to set cli args in UI. So my changes are meaningless. I'll drop them